bin/tronctl

```diff
@@ -110,17 +121,42 @@ def control_objects(args):
             possibilities = list(tron_jobs_completer(prefix='', client=tron_client))
             suggestions = suggest_possibilities(word=identifier, possibilities=possibilities)
             raise SystemExit(f"Error: {e}{suggestions}")
-
         yield edit(args, tron_id.url)
 
 
+def backfill(args):
+    if args.start_date is None:
+        print("Error: For a backfill, --start-date must be set")
+        return 1
+    if args.end_date is None:
+        args.end_date = datetime.datetime.today()
```

Comment:
Will this be a little too dangerous? If someone forgot to set end_date, this would backfill more than he/she wants.

Reply:
I think it is a sane default, plus with the manual verification before you run something, we are always giving users a second chance to review what is about to happen.

Reply:
Ok, fair enough.